### DISCUSSION TITLE
post/create-configure-vm-azure-cli

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#14] [Announcements]
# post/create-configure-vm-azure-cli

Mark Heath's Development Blog

https://markheath.net/post/create-configure-vm-azure-cli

**Comment by Callaghana** [comment]
Much appreciated.  Would you use convertTo-JSON to turn your PS script to JSON?

**Reply by markheath** [reply]
I'm not entirely sure what you're asking, but convertTo-JSON is useful if you have a PowerShell object that you want to format as JSON (maybe to pass to an Azure CLI command)
